persistence/soft-index/src/main/java/org/infinispan/persistence/sifs/SoftIndexFileStore.java

```diff
@@ -150,7 +151,7 @@ public void start() {
       temporaryTable = new TemporaryTable(configuration.indexQueueLength() * configuration.indexSegments());
       storeQueue = new SyncProcessingQueue<>();
       indexQueue = new IndexQueue(configuration.indexSegments(), configuration.indexQueueLength());
-      fileProvider = new FileProvider(getDataLocation(), configuration.openFilesLimit());
+      fileProvider = new FileProvider(getDataLocation(), configuration.openFilesLimit(), "ispn.");
```

Comment:
It would be good to have the prefix values as constants, e.g. 10_1_PREFIX = "", 11_0_PREFIX="ispn."

Reply:
:+1: